core/2d/CCDrawNode.cpp

```diff
@@ -369,8 +369,8 @@ void DrawNode::drawCircle(const Vec2& center,
     }
     if (_lineWidth > minThickness)
     {
-        auto scaleFactor = AX_CONTENT_SCALE_FACTOR() * 2 ;
-        drawPolygon(vertices, segments, Color4B(1.0f, 0.0f, 0.0f, 1.0f), _lineWidth / scaleFactor, color);
+        static GLView* glView = _director->getOpenGLView();
```

Comment:
please don't use static

Reply:
I used it for this reason:
glView will be "only one" initialized (not every call) => faster and it will be done only if the _lineWidth > minThickness.